premerge/gke_cluster/main.tf

```diff
@@ -78,6 +78,37 @@ resource "google_container_node_pool" "llvm_premerge_linux" {
   }
 }
 
+resource "google_container_node_pool" "llvm_buildbot_linux" {
```

Comment:
In my opinion, the names should mention postsubmit, to make it very clear what these are for.

Reply:
To me 'buildbot' just meant a machine/configuration that builds & runs tests. pre vs post never came into it.  If you don't want to change the names, how about adding a comment pointing out that all the 'buildbot' things refer to postcommit building/testing?

Reply:
Added a comment.

Reply:
Thanks!